x11-wm/bspwm/bspwm-0.9.12.ebuild

```
@@ -0,0 +1,45 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

```suggestion
EAPI=8
```
Always aim to use the latest available EAPI, 8 in this case.